packages/ai-twelvelabs/tests/text-adapter.test.ts

```
@@ -0,0 +1,186 @@
+
+const analyzeStreamMock = vi.fn()
+const analyzeMock = vi.fn()
+
+vi.mock('twelvelabs-js', () => ({
+  TwelveLabs: class {
+    analyzeStream = analyzeStreamMock
+    analyze = analyzeMock
+  },
+}))
+
+import { createTwelveLabsText } from '../src/adapters/text'
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)